content/automate/ManageIQ/Transformation/Common.class/__methods__/waitforhandover.rb

```diff
@@ -14,6 +14,8 @@ def main
               @handle.root['ae_result'] = 'retry'
               @handle.root['ae_retry_server_affinity'] = true
               @handle.root['ae_retry_interval'] = 15.seconds
+            elsif @task.get_option(:progress)[:status] == "error"
```

Comment:
Does this check happen only to `@task.state = 'migrate'` or `@task.get_option(:workflow_runner) = 'automate'`?

Reply:
Nope it has to be both `@task.state = 'migrate'` _and_ `@task.get_option(:workflow_runner) = 'automate'`. Fixed the specs.